kin-sdk-core/src/main/java/kin/sdk/core/AccountActivator.java

```diff
@@ -56,6 +51,21 @@ void activate(@NonNull Account account, @NonNull String passphrase) throws Opera
         }
     }
 
+    private void verifyParams(@NonNull Account account, @NonNull String passphrase) {
```

Comment:
You can use as `verifyParams(@NonNull Object... objs)` and do a for loop of checknotnull

Reply:
I won't be able to infer the param name if I switch to loop (I need it for proper exception message)